Engine/ScriptAnalyzer.cs

```diff
@@ -1644,6 +1646,31 @@ private static Encoding GetFileEncoding(string path)
             }
         }
 
+        /// <summary>
+        /// Inspects Parse errors and removes TypeNotFound errors that can be ignored that can be due to types that are not known yet (e.g. due to 'using' statements)
```

Comment:
Mistake in  sentence.

Maybe something like: 
`/// Inspects Parse errors and removes TypeNotFound errors that can be ignored since some types are not known yet (e.g. due to 'using' statements)`